css-canvas-medical/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="row inline-row">
+      <div class="column-header">
+        <img src="images/logo-blue.svg" alt="logo" class="logo-img">
+      </div>
+      <ul class="header-links">
+        <li><a href="">Product</a></li>
+        <li><a href="">Implementation</a></li>
+        <li><a href="">About Us</a></li>
+        <li><a href="">Blog</a></li>
+        <li><button class="contact-us-btn">Contact Us</button></li>
+      </ul>
+    </div>
+    <div class="row justify-center">
+      <div class="hero positioned">
+        <div class="hero-image"></div>
+        <div class="overlay"></div>
+        <div class="column-two-thirds">
+          <h2 class="text-left">The EMR Reimagined for </br>Primary Care</h2>
+          <p class="text-left">Can't remember the last time you talked with a patient without focusing on the EMR?
+            Canvas lets you rediscover joy in the work you do. Capture meaningful data, identify gaps in care
+            , and move seamlessly between tasks within the platform. You stay present with patients while your whole
+            care team stays up-to-speed. Be a better doctor with Canvas, designed for primary care providers with the
+            help of primary care providers.
+          </p>
+          <button class="get-started-btn position-left"> Get Started</button>
+        </div>
+      </div>
+    </div>
+    <div class="row justify-center margin-top background-gray">
+      <div class="column-whole">
+        <h2>Clinicians Prefer Canvas</h2>
+        <p>Physician-owned primary care practice across the country have already </br>
+          made Canvas the fabric of their care.
+        </p>
+      </div>
+    </div>
+    <div class="row justify-center background-gray padding-bottom">
+      <div class="column-half">
+        <p>Cascade Family Practice, Portland OR</p>
+        <p>Temescal Creek Medicine, Oakland Ca</p>
+        <p>San Juan Healthcare, Friday Harbor WA</p>
+      </div>
+      <div class="column-half">
+        <p>Ward Medical Clinic, Bonifay Fl</p>
+        <p>Caribou Medical, Towanda PA</p>
+        <p>Bay Tree Family Care, Red Bay AL</p>
+      </div>
+    </div>
+    <div class="row space-between align-center">
+      <div class="column-half-spacing">
+        <h2 class="tiempos dark-gray">Ready to learn more?</h2>
+        <p class="avenida dark-gray">Schedule a private meeting now</p>
+      </div>
+      <div class="column-half-spacing">
```

Comment:
instead of `column-half-spacing`, use `column-half` and make a second class to handle whatever spacing you are doing.